client/my-sites/sidebar-unified/item.jsx

```diff
@@ -33,21 +33,25 @@ export const MySitesSidebarUnifiedItem = ( {
 	slug,
 	title,
 	url,
+	continueInCalypso,
```

Comment:
Shall we get this added to `propTypes` since we're executing it `onNavigate` every time?

Reply:
Done 923686b!